doc/ansible/user-guide.md

```diff
@@ -256,23 +256,37 @@ Once this is done, there are two ways to run the operator:
 
 Running as a pod inside a Kubernetes cluster is preferred for production use.
 
+Setup the DOCKER_IMAGE environment variable corresponding to your quay registry:
+```
+$ export QUAY_USERNAME="put your quay username here"
+$ export DOCKER_IMAGE="quay.io/${QUAY_USERNAME}/memcached-operator:v0.0.1"
+```
+
 Build the memcached-operator image and push it to a registry:
+
 ```
-$ operator-sdk build quay.io/example/memcached-operator:v0.0.1
-$ docker push quay.io/example/memcached-operator:v0.0.1
+$ operator-sdk build $DOCKER_IMAGE
+$ docker login quay.io
```

Comment:
It is required be done just once. 
So, for example, I already have this config then I do not really need to do it.

Reply:
Sure, so I should just remove line 268?

Reply:
Following the clarifications
-  `$ docker login quay.io` -> The login not necessary will be to quay.io and it may not required for all since the user may have this config. So, IMHO it could be a NOTE such as:

> **NOTE**: Ensure that you are logged in your registry. (E.g `docker login quay.io`) 
 
- `$ operator-sdk build $DOCKER_IMAGE` -> It is the command that does the build then it should be kept. 
- `$ docker create $DOCKER_IMAGE`-> It should be removed since it is not required at all. 
- `$ docker push $DOCKER_IMAGE`-> It should be kept